dagstore.go

```diff
@@ -98,9 +98,8 @@ type DAGStore struct {
 
 	// Throttling.
 	//
-	throttleFetch throttle.Throttler
-	throttleCopy  throttle.Throttler
-	throttleIndex throttle.Throttler
+	throttleFetchCopy throttle.Throttler
```

Comment:
I think we can just call this `throttleFetch` (as copy is part of the fetch operation).